style.css

```diff
@@ -1,6 +1,1175 @@
 
 @import url('https://fonts.googleapis.com/css?family=Poppins');
 
+@media screen and (min-width: 769px) {
+  .ul {
+    list-style: none;
+    padding-left: 0;
+  }
+
+  .ul1 {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .nav {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 80%;
+    z-index: 1;
+    padding-left: 10%;
+    padding-right: 10%;
+  }
+
+  .nav-li5 {
+    display: none;
+  }
+
+  .nav-li1 {
+    width: 78%;
+  }
+
+  .nav-li2,
+  .nav-li3,
+  .nav-li4 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    margin-right: 60px;
+  }
+
+  .nav-li1 a {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    color: #6070ff;
+    text-decoration: none;
+  }
+
+  .div {
+    width: 25px;
+    height: 2.5px;
+    margin: 3px;
+    border-radius: 5px;
+    background: #6070ff;
+  }
+
+  .text-section {
+    background-image: url('img/Headerbg.png');
+    background-size: 100% 100%;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 12px;
+    border-bottom-left-radius: 90px;
+    box-shadow: 0 0 0 150px #e5e5e5;
+    background-position: bottom;
+    height: 916px;
+    padding-left: 10%;
+    padding-right: 10%;
+  }
+
+  .box {
+    align-self: flex-start;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-right: 0;
+  }
+
+  .p1 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    width: 80%;
+  }
+
+  .h1 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 60px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    letter-spacing: 0.37px;
+    width: 500px;
+  }
```

Comment:
- [ ] You are still not using the right values as specified in the design. Please make sure you apply the right font size and line heights on this text.

  ![Isaac1](https://user-images.githubusercontent.com/87394385/213795037-b1431522-5a82-43d2-a941-b579cc53d51c.JPG)

-   Please also note that the heading should break after your name and 'Glad' should start on the following line. ou can implement this by using line breaks like this : `<hx>I’m Isaac <br>Glad to see you!</hx>`